features/.nav

```diff
@@ -44,22 +44,21 @@
   - color.feature
   - fail_fast.feature
   - custom_settings.feature
-  - alias_example_to.feature
   - default_path.feature
 - expectation_framework_integration:
-  - configure_expectation_framework.feature
   - failure_aggregation.feature
-- mock_framework_integration:
-  - use_rspec.feature
-  - use_flexmock.feature
-  - use_mocha.feature
-  - use_rr.feature
-  - use_any_framework.feature
 - formatters:
   - text_formatter.feature
   - configurable_colors.feature
   - json_formatter.feature
   - custom_formatter.feature
-- spec_files:
-  - arbitrary_file_suffix.feature
 - core_standalone.feature
+- legacy_support_and_special_project_needs
```

Comment:
Done. 

I chose "alternate tools and libraries", so it sounds general enough for people to know to look there if needed, while still having a single category.

Without much thinking, I lazily converted the feature file into an integration tests - I'm sure you'll have lots of comments, so fire away.

I moved the file_suffix and alias_example into a separate category: "legacy_features_and_workarounds".